estilo.css

```
@@ -0,0 +1,3 @@
+* {
+  color: red;
```

A cor vermelho não ficou legal, acho melhor usar laranja, pois o tom tem um sentido de passar tranquilidade.
```suggestion
  color: orange;
```